### ISSUE TITLE
Follow UNet paper for training specifics

### ISSUE THREAD

**Issue by thomashopkins32** [thomashopkins32/HuBMAP#35]
For the first couple of releases, I trained the model using things from my personal experience and intuition in training deep neural networks. I should look to see what worked for the authors and try to emulate that for this dataset.

This means that I should:
- Use SGD with momentum
- Use xavier normal weight initialization (check the math for this)
- Use data augmentation similar to their choices
- Use dropout regularization near the final layers of the network

Let's see if that will improve our performances.

**Comment by thomashopkins32** [comment]
Next step is to visually confirm that the elastic transforms work.

Then we need to implement the pixel weighting scheme based on class frequencies.
